Modules/IO/Meta/src/itkMetaImageIO.cxx

```diff
@@ -637,7 +637,7 @@ MetaImageIO::WriteImageInformation()
       unsigned int i = 0;
       while (i < vval.size() - 1)
       {
-        strs << vval[++i] << ' ';
+        strs << vval[i++] << ' ';
```

Comment:
I prefer a function call like:
```c++
// Function to join strings with a delimiter similar to python's   ' '.join([1, 2, 3 ])
void join(const std::vector<std::string>& elements, const std::string& delimiter,  std::ostringstream strs) {
    for (size_t i = 0; i < elements.size(); ++i) {
        strs << elements[i];
        if (i != elements.size() - 1) {
            strs << delimiter;
        }
    }
}
```

This avoids the pre-post increment relationship to the [] operator.